CONTRIBUTING.md

```diff
@@ -73,6 +73,15 @@ The documentation is located in the `website` directory and can be started with
 
 There are other available commands too. As set up in the [.build](./.build/) directory.
 
+## How to Check the docs
+
+We use [Markdownlint](https://github.com/DavidAnson/markdownlint) to check markdown formatting and [cSpell](https://cspell.org) for spelling. We have GitHub actions to check these for PRs, but if you'd like to check locally, you can:
+
+- Install cSpell: `npm install -g cspell`
+- Install the markdownlint CLI: `npm install -g markdownlint-cli`.
```

Comment:
ℹ️ Similar to the above, I can remove the markdown lint related text if desired prior to merge.